lib/transports/ws2.js

```diff
@@ -2214,24 +2228,26 @@ class WSv2 extends EventEmitter {
 
   /**
    * @param {Object} opts
-   * @param {string} opts.symbol
+   * @param {string?} opts.pair
+   * @param {string?} opts.symbol
    * @param {string} opts.cbGID - callback group id
    * @param {Method} cb
    * @see https://docs.bitfinex.com/v2/reference#ws-auth-funding-trades
    */
-  onFundingTradeEntry ({ symbol, cbGID }, cb) {
-    this._registerListener('fte', { 1: symbol }, FundingTrade, cbGID, cb)
+  onFundingTradeEntry ({ pair, symbol, cbGID }, cb) {
+    this._registerListener('fte', { 0: pair || symbol }, FundingTrade, cbGID, cb)
```

Comment:
Why use pair here? I thought a funding trade only returned a single symbol i.e `fUSD`?

Reply:
Good point; I'm not sure what I was thinking, refactored it now to use only `symbol`